firebase-perf/src/test/java/com/google/firebase/perf/application/AppStateMonitorTest.java

```diff
@@ -15,6 +15,7 @@
 package com.google.firebase.perf.application;
```

Comment:
The behavior of AppStateMonitor has changed quite a bit. I'm surprised the tests haven't changed barely at all. I feel like we're probably missing tests (e.g. testing new behaviors), as well as having coverage for the behavior changes that have happened (some things happen in different points of the activity's lifecycle)

Reply:
The only lifecycle change is reverting the activity screen trace that is from `onResume` -> `onPostPaused` to `onStart` -> `onStop`. `onStart` -> `onStop` was the original behavior before attempting to fix the b/210055697 that started all of this in the first place. That's why there's not new tests for that.